connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorHeartbeatConfig.java

```diff
@@ -77,7 +79,9 @@ short heartbeatsTopicReplicationFactor() {
                     ConfigDef.Importance.LOW,
                     HEARTBEATS_TOPIC_REPLICATION_FACTOR_DOC);
 
+    protected final static ConfigDef CONNECTOR_CONFIG_DEF = new ConfigDef(mergeConnectorConfigDef(HEARTBEAT_CONFIG_DEF));
```

Comment:
With the above suggestion, this can now become:
```java
protected static final ConfigDef CONNECTOR_CONFIG_DEF = defineHeartbeatConfig(new ConfigDef(BASE_CONNECTOR_CONFIG_DEF));
```